rotkehlchen/data_import/importers/rotki_events.py

```diff
@@ -50,8 +50,10 @@ def _consume_rotki_event(
         location = Location.deserialize(csv_row['Location'])
         timestamp = TimestampMS(deserialize_timestamp(csv_row['Timestamp']))
         fee = Fee(deserialize_asset_amount(csv_row['Fee'])) if csv_row['Fee'] else Fee(ZERO)  # noqa: E501
+        asset_resolver = LOCATION_TO_ASSET_MAPPING.get(location, asset_from_common_identifier)
+        asset = asset_resolver(csv_row['Currency'])
```

Comment:
naming nitpick, since at least for me (and the way we use classes in the code) mapping, sounds like a dict mapping, while resolver like a class

```suggestion
        asset_mapping = LOCATION_TO_ASSET_MAPPING.get(location, asset_from_common_identifier)
        asset = asset_mapping(csv_row['Currency'])
```